app/src/main/java/org/sil/storyproducer/controller/logging/LogListAdapter.kt

```diff
@@ -34,8 +34,8 @@ internal class LogListAdapter(private val context: Context, slide: Int) : BaseAd
         for (le in allEntries) {
             when(le.phase.phaseType){
                 PhaseType.LEARN -> if(learn) displayEntries.add(le)
-                PhaseType.DRAFT -> if(draft) displayEntries.add(le)
-                PhaseType.COMMUNITY_CHECK -> if(comCheck) displayEntries.add(le)
+                PhaseType.TRANSLATE_REVISE -> if(draft) displayEntries.add(le)
+                PhaseType.COMMUNITY_WORK -> if(comCheck) displayEntries.add(le)

```

Comment:
draft and comCheck parameters could get updated names

Reply:
done